.gitpod.yml

```diff
@@ -1,26 +1,38 @@
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - name: Uninstall lynx
-    init: sudo apt-get remove lynx -y
-  - name: Install AWS CLI
+  - name: Gitpod config (browser open, workspace bin path)
     init: |
-      curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-      unzip awscliv2.zip
-      sudo ./aws/install
-      rm awscliv2.zip
-      rm -rf ./aws
-    command: aws --version
-
-  - name: Install Conveyor CLI
+      mkdir -p /workspace/bin
+      cat > /workspace/bin/open.sh <<'EOF'
+      #!/bin/bash
+      exec gp preview --external "$@"
+      EOF
+      chmod +x /workspace/bin/open.sh
+    command: |
+      sudo update-alternatives --install /usr/bin/www-browser www-browser /workspace/bin/open.sh 100
+      echo "export PATH=\"/workspace/bin:\$PATH\"" >> ~/.bashrc
+      exit
+  - name: Install conveyor
     init: |
+      tmpdir="$(mktemp -d)"
```

Comment:
Just a random thought, don't know if you know the answer. Is it possible to download the latest version of the CLI whenever gitpod starts up?

Reply:
Yes! If we just move this piece of code from `init:` to `command:` it will execute at every startup. In that case, you can also install to `/usr/local/bin`or similar instead of to the `/workspace` volume.

Do you think this behaviour is better? I can see how it might be good for `conveyor`, but when installing "everything" at startup, starting workspaces gets a bit slow, so I like using prebuilds/init.